_publications/alsaadi2020comparing.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Comparing Workflow Application Designs for High Resolution Satellite Image Analysis"
+collection: publications
+permalink: /publications/alsaadi2020comparing
+date: 2020-10-27
+type: pub
+author: "Aymen Al-Saadi, Ioannis Paraskevakos, Bento Collares Gonçalves, Heather J. Lynch, Shantenu Jha and Matteo Turilli"
+venue: "arxiv"
+paperurl: N/A
+arxiv: https://arxiv.org/pdf/2010.14756.pdf
```

Comment:
Instead of the pdf link to the pub's page.

Reply:
This URL still points to the PDF file of the publication on Arxiv. Can you change it to https://arxiv.org/abs/2010.14756?

Reply:
This is still pending